operator/src/main/resources/example-postgres.yaml

```diff
@@ -0,0 +1,39 @@
+# PostgreSQL StatefulSet
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: postgresql-db
```

Comment:
should we always prefix the operands with the cr name ? or at least with keycloak ? In the same namespace the user could have another postgresql-db for other purposes. Wdyt ?

Reply:
Yes you are right....I see what you mean.